x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/types.ts

```diff
@@ -114,7 +114,7 @@ export interface EndpointIndexUIQueryParams {
   /** Which page to show */
   page_index?: string;
   /** show the policy response or host details */
-  show?: 'policy_response' | 'activity_log' | 'details' | 'isolate';
+  show?: 'policy_response' | 'activity_log' | 'details' | 'isolate' | 'unisolate';
```

Comment:
can this type definition and the `allowedShowValues` be combined into a single enum somehow?

So that the `show?` type here is still restricted to those specific values, and (hopefully) that enum (or whatever appropriate TS concept would accomplish the same) can also be used in that check above in `selectors.ts`? Basically so the string values are just listed out once.

Reply:
Yeah, I think thats a good idea. `Enum` might create a problem where we are using `string` to assign the value today and thus it might require a bit of refactoring. I'll keep this in mind during my next PR.